SlackerReferenceApp/bower.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "HelloIonic",
+  "private": "true",
+  "devDependencies": {
+    "ionic": "driftyco/ionic-bower#1.1.1"
```

Comment:
Bower was installed by Ionic, so I'm not sure if I get this dependency here. I can play around with removing it, if you think that is best